ua-snap/swti
# Updates underlying normals to 1991-2020

To test this:

 1. Run existing code in `main` in production mode (do not set `FLASK_DEBUG`)
 2. Download a snapshot of the resulting view.
 3. Switch to branch, restart server, reload page.

Image from before update:

![Unknown-2](https://user-images.githubusercontent.com/525049/135927420-45125b28-fa88-4be0-8d16-11f09f2409db.png)

Image from after update:

![Unknown-3](https://user-images.githubusercontent.com/525049/135927434-70644b46-7909-475d-bf12-b1f0889d5a05.png)

Everything else should be the same.
